### ISSUE TITLE
LibWeb: Input elements with placeholder text are doubly tall suddenly

### ISSUE THREAD

**Issue by awesomekling** [SerenityOS/serenity#17994]
Example from GitHub:

![image](https://user-images.githubusercontent.com/5954907/227028322-2b07186a-3109-49b5-8ad0-2f23e2f2df9c.png)

Likely regression from recent changes to add `lh` CSS units.

cc @skyrising

**Comment by skyrising** [comment]
Yes, the problem is that the placeholder and the content nodes are both visible. In Chromium they are both represented as elements inside the shadow root, while in LibWeb the content is an element and the placeholder just a layout node. I haven't yet figured out how Chromium positions them (since only the placeholder is conditionally hidden, the content is always visible). I'll investigate more in the next couple days.

**Comment by skyrising** [comment]
Fixed by 4df6bcd906c91cabcd695f0657ed77b97cafaf9b
